### PR TITLE
Fix Nan loss skipping logic

### PR DESCRIPTION
**Pull request by chaitjo**
I think that the previous version of the code was detecting a nan/inf loss value but would still add that value to the aggregated overall loss.

This PR fixes the issue.

### CODE REVIEW

File: protenix/model/loss.py [hunk 1]
```diff
@@ -1555,7 +1553,11 @@ def aggregate_losses(
                 all_metrics[loss_name] = loss.detach().clone()
                 all_metrics[f"weighted_{loss_name}"] = weight * loss.detach().clone()
 
-            cum_loss = cum_loss + weight * loss
```
Comment (zhangyuxuann) [inline protenix/model/loss.py:1558]: There's no need to change this part. Previously, we found that cases with NaN losses were all caused by bad data — for example, sequences that are entirely unresolved. Such cases should be skipped directly at the data level.

Moreover, keeping the cumulative loss (cum_loss) as NaN when invalid allows us to dump the corresponding batch information for further analysis. This is helpful for debugging and identifying problematic samples. With the current approach, cum_loss is always a valid value, which makes it harder to trace such issues. Could you please create a separate PR for the typo fixed change configs/configs_model_type.py?
